src/prototype_room_my.js

```diff
@@ -262,7 +262,7 @@ Room.prototype.executeRoom = function() {
       amount = 2;
       // TODO maybe better spawn harvester when a carry recognize that the dropped energy > threshold
       if (room.controller.level == 2 || room.controller.level == 3) {
-        amount = 5;
+        amount = 3;
```

Comment:
I think this should be removed and the carry should call harvesters on RCL < 4, when dropping energy and dropped energy > threshold.

But fine for now